pkg/storage/storage_test.go

```diff
@@ -0,0 +1,327 @@
+package storage
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+)
+
+func TestStorage_AddProfile(t *testing.T) {
+	tests := []struct {
+		name       string
+		storage    *Storage
+		newProfile *Profile
+		validate   func(*testing.T, *Storage)
+	}{
+		{
+			name: "add new Profile",
+			storage: &Storage{
+				Default: "default",
+				Profiles: []*Profile{
+					{Name: "default", Key: "key"},
+				},
+			},
+			newProfile: &Profile{Name: "test", Key: "key"},
+			validate: func(t *testing.T, s *Storage) {
+				if len(s.Profiles) != 2 {
+					t.Fatalf("got %d profiles, want 2", len(s.Profiles))
```

Comment:
Please check https://github.com/stretchr/testify it contains amount of handy methods to verify stuff

So here it will be just like:

```
r.Len(s.Profiles, 2)
r.Equal([]Profiles{
  {
    Name: "test",
  },
}, s.Profiles)
```